cmd/lncli/cmd_macaroon.go

```diff
@@ -190,3 +190,24 @@ func bakeMacaroon(ctx *cli.Context) error {
 
 	return nil
 }
+
+var listMacaroonIDsCommand = cli.Command{
```

Comment:
Same here: I'd merge all changes to the `lnrpc` package and to `rpcserver.go` into one commit and all changes to `lncli` into another commit. I get that you wanted to separate by new command/functionality. But we prefer to have all changes to one unit/package put together if possible. That way the commit message prefix can name the changed unit/package.

Reply:
np, I'll change the commits based on uint/package.